lib/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -11,6 +11,33 @@ module Helpers
             sections only: %w[all]
           end
 
+          apipie :method, "Returns next integer stored in a hostgroup param named 'sequence_num', starts and increments it by a given amount" do
```

Comment:
Doesn't it return a string with next integer?

Reply:
Yeah, technically it's integer in string (because of formatting), I will reword this.